dev/source/docs/ros2-pi.rst

```diff
@@ -95,3 +95,183 @@ The simplest solution is to wire an open telemetry port.
 
 .. figure:: ../images/PiZero2WTelem.png
    :target: ../_images/PiZero2WTelem.png
+
+..
+    Cosider documenting flow control setup, and also any configuration to enable /dev/ttyS0
+
+This allow GPIO13 and GPIO15 (/dev/ttySO) to communicate with ArduPilot.
+
+Cross Compile an application with Docker
```

Comment:
Make this section optional, with a note saying that it is only required for low memory devices (such as the Pi Zero 2). Then add a section for local compilation.